src/domUpdates.js

```diff
@@ -109,7 +110,7 @@ let domUpdates = {
     }).join(", ");
   },
 
-  generateInstructions(recipe, fullRecipeInfo) {
+  displayInstructions(recipe, fullRecipeInfo) {
```

Comment:
Nice. I like the changed name because now we know exactly what this DOM function does

Reply:
I concur. Good change for what it does.